sdk/src/beta9/runner/common.py

```diff
@@ -391,6 +397,8 @@ def __exit__(self, *_, **__):
 
 
 def wait_for_checkpoint():
+    initialize_workers_ready()
```

Comment:
I'm not sure we should do it like this, because then we're creating the Value in the subprocess vs before it launches and I don't think it will have the same effect. We could just check if we're in the remote environment and only initialize it if so maybe

Reply:
I can make that change as well. However, it doesn't seem like the Value is used anywhere else besides `wait_for_checkpoint`. Does it even need to be global (not sure I totally follow its usage)?

Reply:
It's used to synchronize worker processes as they are spinning up before auto checkpoint. I think it has to either be global or passed into the worker processes when they start (either would work, global was simpler)

Reply:
Ahh got it. Updated to use `is_remote` check instead of the current method